### PR TITLE
Update api docs

### PR DESCRIPTION
**Pull request by MikaelSmith**
Add API docs for the /fs/schema endpoint and improve /fs/exec docs.

### CODE REVIEW

File: api/schema.go [hunk 1]
```diff
@@ -8,6 +8,22 @@ import (
 	"github.com/puppetlabs/wash/plugin"
 )
 
+// swagger:route GET /fs/schema schema entrySchema
+//
+// Schema for an entry at path
+//
+// Returns an EntrySchema object describing the plugin schema starting at the given path.
```
Comment (ekinanp) [inline api/schema.go]: Looking at some other types (e.g. `execBody/execResponse`), I think you can document this as `map[string]*EntrySchema` noting that the first key in the map corresponds to `<path>`'s schema.

Comment (MikaelSmith) [reply api/schema.go]: Ah, because that's how it'll be marshaled?

Comment (MikaelSmith) [reply api/schema.go]: And are those apitypes.EntrySchema or plugin.EntrySchema?

Comment (MikaelSmith) [reply api/schema.go]: Looks kind of challenging to document.

**Comment by MikaelSmith** [comment]
What it looks like now:
Screen Shot 2019-07-15 at 2 40 51 PM

**Pull request merged**